configs/openSUSE/dbus-services.toml

```diff
@@ -1130,6 +1130,21 @@ path = "/usr/share/dbus-1/system-services/org.freedesktop.home1.service"
 digester = "shell"
 hash = "c4281ad74af8c43a963806142e2770751b153df8c1217da4edf212b334ecddcf"
 
+# TODO: merge with entry above once v258 reaches Factory
+[[FileDigestGroup]]
+package  = "systemd-homed"
+note     = "systemd-homed v258 changes"
+bug      = "bsc#1250884"
```

Comment:
This also looks like the wrong bsc# reference, both in the commit message and the changes